src/test/java/dog/pawbook/logic/parser/PawbookParserTest.java

```diff
@@ -83,12 +85,64 @@ public void parseCommand_list() throws Exception {
 
     @Test
     public void parseCommand_unrecognisedInput_throwsParseException() {
-        assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE), ()
-            -> parser.parseCommand(""));
+        assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE), (
+        ) -> parser.parseCommand(""));
```

Comment:
The brackets of the the lambda arguments should not be split into 2 lines, especially not when they are empty.

Reply:
Yea, I know but the codestyle forces me to do so. Idk why lol

Reply:
I noticed that too

Reply:
checkstyle has passed for these code or else it won't have been possible to merge them, so I think it might be a problem with your IDE configuration.

Reply:
OK, I will check that again. Coz it shows an error of separator wrap which is regarding the brackets.